packages/deployments/contracts/contracts/core/messaging/connectors/PolygonConnector.sol

```diff
@@ -0,0 +1,101 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity 0.8.15;
+
+import {IRootManager} from "../interfaces/IRootManager.sol";
+import {Connector} from "./Connector.sol";
+import {FxBaseRootTunnel} from "../helpers/polygon/tunnel/FxBaseRootTunnel.sol";
+import {FxBaseChildTunnel} from "../helpers/polygon/tunnel/FxBaseChildTunnel.sol";
+
+// address constant MUMBAI_FX_CHILD = 0xCf73231F28B7331BBe3124B907840A94851f9f11;
+// address constant GOERLI_CHECKPOINT_MANAGER = 0x2890bA17EfE978480615e330ecB65333b880928e;
+// address constant GOERLI_FX_ROOT = 0x3d1d3E34f7fB6D26245E6640E1c50710eFFf15bA;
+
+// address constant MAINNET_FX_CHILD = 0x8397259c983751DAf40400790063935a11afa28a;
+// address constant MAINNET_CHECKPOINT_MANAGER = 0x86E4Dc95c7FBdBf52e33D563BbDB00823894C287;
+// address constant MAINNET_FX_ROOT = 0xfe5e5D361b2ad62c541bAb87C45a0B9B018389a2;
+
+contract PolygonL2Connector is Connector, FxBaseChildTunnel {
+  // ============ Constructor ============
+  constructor(
+    uint32 _domain,
+    uint32 _mirrorDomain,
+    address _amb,
+    address _rootManager,
+    address _mirrorConnector,
+    uint256 _mirrorProcessGas,
+    uint256 _processGas,
+    uint256 _reserveGas
+  )
+    Connector(_domain, _mirrorDomain, _amb, _rootManager, _mirrorConnector, _mirrorProcessGas, _processGas, _reserveGas)
+    FxBaseChildTunnel(_amb)
+  {
+    // setFxRootTunnel(_mirrorConnector);
```

Comment:
do we need this still?

Reply:
we don't need when deploying, but we should have extra tx for set fxRootTunnel and fxChildTunnel while configuring mirror connectors.